### PR TITLE
Highlighting advanced and basic topics in the tutorial

### PR DESCRIPTION
**Pull request by rswamina**
# Pull Request checklist

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes [242](https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/242)

### CODE REVIEW

**Comment by rswamina** [comment]
@SarahAlidoost and @remi-kazeroni : I made a note of basic and advanced topics in the Home page. Not sure if we want it elsewhere. Please advice or if you think this is sufficient, please approve for merge.

File: index.md [hunk 1]
```diff
@@ -11,8 +11,11 @@ software toolkit that aims to facilitate the diagnosis and evaluation of the
 causes and effects of model biases and inter-model spread within the CMIP model
 ensemble.
 
-This tutorial is structured such that the main body of the tutorial, up to the
-episode 7, can be done in one sitting. From episode 8, each episode is a
+This tutorial is structured into ``basic`` and ``advanced`` topics such that episodes 
+starting from the [Introduction]({{ page.root }}{% link _episodes/01-introduction.md %}) up to the episode on [Conclusion of the basic tutorial]({{ page.root }}{% link _episodes/05-conclusions.md %})
+all cover basic topics and can be done in one sitting. 
+
+The advanced topics cover the remaining episodes and each episode is a
```
Comment (remi-kazeroni) [inline index.md:18]: Perhaps, this could also be written as: 
```suggestion
The remaining episodes cover the advanced topics and each episode is a
```

**Review by remi-kazeroni — APPROVED** [review]
Thanks @rswamina! I think this is the right place to mention the difference between basic and advanced topics.

**Comment by rswamina** [comment]
@SarahAlidoost - Do you want to take a look at the changes before merging?

**Review by SarahAlidoost — APPROVED** [review]
@rswamina thanks :+1:

**Pull request merged**